styles.css

```diff
@@ -330,9 +318,11 @@ a {
   margin: 7px;
 }
 
-/* mobile view */
-
-@media (max-width: 710px) {
+/*
+---------------------------------------------------
+mobile view
+-------------------------------------------------- */
+@media (max-width: 600px) {
```

Comment:
- [OPTIONAL] Comments makes the code messy and the code should be understandable by itself. Try removing commented code.
 - [OPTIONAL] You are developing a mobile-first application so you should apply all CSS for mobile before `@media (min-width: ...px)`.